index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <title>Joseph Kwesi Kwentsir Portfolio</title>
+</head>
+<body>
+    <header class="page-header">
+      <nav class="nav-header"> 
+          <ul>
+              <li>Welcome</li>
+              <li> <a href="#"> <img src="/images/Union.svg" alt=""/></a></li>
+          </ul>
+      </nav>
+    </header>
+    
+
+      <section class="page-headline">
+        <h2 class="intro">
+            Hey there. I’m Joseph
+            <br>
+            <span class="job">I’m a software developer</span>
+        </h2> 
```

Comment:
- [ ] Please change the `h2` tag into an `h1` and the span to an h2 tag instead, you should at least have one h1 title in your web app.